pkg/controller/run/main.go

```diff
@@ -153,6 +153,10 @@ type Action struct {
 }
 
 func (ctrl *Controller) parseLine(ctx context.Context, logE *logrus.Entry, line string, cfg *Config) (string, error) { //nolint:cyclop,funlen
+	// Eliminate single/double quotes from the line
+	line = strings.ReplaceAll(line, "'", "")
+	line = strings.ReplaceAll(line, `"`, "")
```

Comment:
Hmm. I think pinact should keep quotes ideally.

Reply:
Quotes are available in a Git tag...

```
git tag "v2'"
git tag 'v2"'
```

Reply:
I'm sorry I created this PR without any discussions😓
I don't know we can use quotes in git tag...

I found we can enclose actions with quotes (e.g. `- use: 'example/action@'`) and found the pinact cannot handle those cases.
But it seems thinking about quotes specifications related to yaml and git tag version is a little bit complicated. And we could need more discussions.

Is it okay to close this PR?

Reply:
Yeah, it's okay to close.
We can't merge this pr, but I think you need this change for your work and I don't want to block it.
So I published a prerelease version v0.1.3-1.

https://github.com/suzuki-shunsuke/pinact/releases/tag/v0.1.3-1

Reply:
Q: Instead of `strings.ReplaceAll`, regex pattern would be possible to use to match the whole expression since important part is quotation at the beginning and at the end?